### PR TITLE
ticdc: fix some error 

### PR DESCRIPTION
**Pull request by asddongmen**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: ticdc/ticdc-sink-to-mysql.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ Sink URI 用于指定 TiCDC 目标系统的连接信息，遵循以下格式：
 一个通用的配置样例如下所示：
 
 ```shell
---sink-uri="mysql://root:123456@127.0.0.1:3306/?worker-count=16&max-txn-row=5000&transaction-atomicity=table"
+--sink-uri="mysql://root:123456@127.0.0.1:3306/?worker-count=32&max-txn-row=2048"
```
Comment (nongfushanquan) [inline ticdc/ticdc-sink-to-mysql.md:44]: ?worker-count=32&max-txn-row=2048"  这两个参数需要配置吗，默认是不是不用配置？

Comment (asddongmen) [reply ticdc/ticdc-sink-to-mysql.md:44]: 嗯嗯 默认不用配置

File: ticdc/deploy-ticdc.md [hunk 2]
```diff
@@ -131,7 +131,7 @@ tiup cluster upgrade <cluster-name> <cluster-version> --transfer-timeout 600
       pump: {}
       drainer: {}
       cdc:
-        gc-ttl: 86400
+        gc-ttl: 172800
     ```
 
     以上把 TiCDC 的 `gc-ttl` 的值设置为 24 小时。
```
Comment (shichun-0415) [inline ticdc/deploy-ticdc.md:137]:
```suggestion
    以上把 TiCDC 的 `gc-ttl` 的值设置为 48 小时。
```

**Review by shichun-0415 — APPROVED** [review]
LGTM

**Review by nongfushanquan — APPROVED** [review]

**Pull request merged**